controller/src/api.py

```diff
@@ -67,33 +68,80 @@ async def get_auth_user(
         return AuthInfo(username="yhaviv@gmail.com", token=token)
 
 
+def send_to_application(path: str, method: str = "POST", request=None, auth=None, **kwargs):
```

Comment:
```suggestion
def _send_to_application(path: str, method: str = "POST", request=None, auth=None, **kwargs):
```

It is an inner function after all.